### ISSUE TITLE
Investigate package installation issue within R workspace

### ISSUE THREAD

**Issue by marjo-luc** [MAAP-Project/ZenHub#423]
From David:

> the problem we are having is in the biomass_harmonization GitLab repo (https://repo.ops.maap-project.org/lduncanson/biomass_harmonization.git). The notebook gedi_calval_map_demo_backup.ipynb uses the cartopy package. This package will install and load in a Python workspace, but not an R workspace.

**Comment by marjo-luc** [comment]
I added these dependencies and was able to address the issue:

conda install -c conda-forge Cython
conda install -c conda-forge/label/cf202003 pygeos
conda install -c conda-forge/label/cf202003 pyproj

The image is building. Waiting on deployment and user to confirm everything looks okay.

**Comment by marjo-luc** [comment]
@bsatoriu I tested my fix by running an r workspace on `https://ade.ops.maap-project.org/ `and running the above commands then running the notebook that had installation issues. Everything worked. I made updates to the r dockerfile and those all went through the ci/cd successfully as well (see [here](https://mas.maap-project.org/root/ade-base-images/-/pipelines/1988)). It looks like the image is tagged as r:latest, but I get this error when I try to start a workspace with it:

> Pulling image "mas.maap-project.org:5000/root/jupyter-image/r:latest"
Failed to pull image "mas.maap-project.org:5000/root/jupyter-image/r:latest": rpc error: code = NotFound desc = failed to pull and unpack image "mas.maap-project.org:5000/root/jupyter-image/r:latest": failed to resolve reference "mas.maap-project.org:5000/root/jupyter-image/r:latest": mas.maap-project.org:5000/root/jupyter-image/r:latest: not found
Error: ErrImagePull
Error: Failed to run the workspace: "Unrecoverable event occurred: 'Failed', 'Failed to pull image "mas.maap-project.org:5000/root/jupyter-image/r:latest": rpc error: code = NotFound desc = failed to pull and unpack image "mas.maap-project.org:5000/root/jupyter-image/r:latest": failed to resolve reference "mas.maap-project.org:5000/root/jupyter-image/r:latest": mas.maap-project.org:5000/root/jupyter-image/r:latest: not found', 'workspacej89x82ce85f23znk.ws-5ddd74cb7f-w24ns'"

Do the ade-image tags not map directly to their jupyter-image counterparts?
